src/pickup.cpp

```diff
@@ -251,6 +251,116 @@ static bool select_autopickup_items( std::vector<item> &here, std::vector<bool>
     return bFoundSomething;
 }
 
+
+// parameter definition - append_fail_msg - any other message you want to append to the error message displayed
```

Comment:
You're inventing a method header comment format here, we generally use javadoc-like header comments since there are a number of tools that know how to extract them.
They look something like:
```
/**
 * This method does bla bla bla.
 * @param append_fail_message does some stuff.
 * @param silent_chk does some other stuff.
*/
```